CMakeLists.txt

```diff
@@ -218,11 +218,12 @@ else()
   )
 endif() # TRITON_PYTORCH_DOCKER_BUILD
 
+# Need to turn off -Werror due to Torchvision vision header extern initialization
```

Comment:
Add "Unfortunately gcc does not provide a specific flag to ignore the specific warning: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=45977"